### DISCUSSION TITLE
How i know if the bot is ready?

### DISCUSSION THREAD

**Discussion by riobits** [telegraf/telegraf#1852] [Q&A] [answered]
In other bot libraries like discord.js, there is a way to determine if the bot is ready by using `bot.on('ready')`. Does Telegraf have a similar feature? If not, could you please tell me how to determine if the bot is ready?

**Comment by MKRhere** [comment, marked as answer]
See https://github.com/telegraf/telegraf/issues/1749#issuecomment-1327203219

A better solution will be added in the future.
